scripts/test-gui/test-config.example.yml

```diff
@@ -83,3 +83,13 @@ watchdog:
   activatePolling: false
   stabilityThreshold: 1000
 checkForBeta: true
+customizeToolbar:
```

Comment:
Do we need these settings in here? This file is primarily useful if some settings should – in testing mode – be different from the other defaults (for example we always activate the debug mode which is off by default).

I don't think those toolbar buttons need to be in here, so to make the PR leaner, it would be best to remove the overrides here.

Reply:
You are right, my original intention was to do this PR without changing the current panel display. For example the new file and previous/next buttons should default to false. Apparently I completely forgot about this. I will remove the settings here.